ui/component/livestreamLink/view.jsx

```diff
@@ -18,9 +18,11 @@ export default function LivestreamLink(props: Props) {
   const [livestreamClaim, setLivestreamClaim] = React.useState(false);
   const [isLivestreaming, setIsLivestreaming] = React.useState(false);
   const livestreamChannelId = (channelClaim && channelClaim.claim_id) || ''; // TODO: fail in a safer way, probably
+  const isChannelEmpty = !channelClaim || !channelClaim.meta || !channelClaim.meta.claims_in_channel;
 
   React.useEffect(() => {
-    if (livestreamChannelId) {
+    // Don't search empty channels
+    if (livestreamChannelId && !isChannelEmpty) {
       Lbry.claim_search({
```

Comment:
Nice.
This might be faster with page_size=1 and no_totals?
@tzarebczan

Reply:
Is there any difference between `claim_list` and `claim_search` ?
https://lbry.tech/api/sdk#claim_list

Reply:
claim list is specifically one's own publishes.

Reply:
page=1 is a given - page_size=1 would limit results. Not sure how big of a deal it would be.

Reply:
Can you publish something while livestreaming ?
I think will be safer to check more than one claim.

Reply:
the "has no source param" ensures it's only the live streams that are returned. And right now we use the latest one you have published.